demosymfonyformsimple/demosymfonyformsimple.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+use PrestaShop\PrestaShop\Adapter\SymfonyContainer;
+
+class DemoSymfonyFormSimple extends Module
+{
+    public function __construct()
+    {
+        $this->name = 'demosymfonyformsimple';
+        $this->author = 'PrestaShop';
+        $this->version = '1.1.0';
+        $this->need_instance = 0;
+
+        $this->bootstrap = true;
+        parent::__construct();
+
+        $this->displayName = $this->trans('Demo of the Symfony-based configuration form', [], 'Modules.DemoSymfonyFormSimple.Admin');
+        $this->description = $this->trans(
+            'Module demonstrates a simple module\'s configuration page made with Symfony.',
+            [],
+            'Modules.DemoSymfonyFormSimple.Admin'
+        );
+
+        $this->ps_versions_compliancy = ['min' => '8.0.0', 'max' => '8.99.99'];
+    }
+
+    public function getContent()
+    {
+        $route = $this->get('router')->generate('demo_configuration_form');
```

Comment:
```suggestion
        $route = $this->get('router')->generate('demo_configuration_form_simple');
```

it won't work without it :D